mix.exs

```diff
@@ -21,7 +21,6 @@ defmodule LocUmbrella.Mixfile do
   end
 
   defp aliases do
-    [compile: ["compile --warnings-as-errors"]]
```

Comment:
question: why remove this? I think it's a useful default to avoid introducing compiler warnings.

Reply:
Hmmm, I don't fully remember why I removed it at this point. I think it was bothering me while I was trying to get the feature working, and I figured that since it's still used in CI https://github.com/mbta/commuter_rail_boarding/blob/213eb4ac72e5c678b06f3298e98c36b9a9dbd1ff/.github/workflows/elixir.yml#L60

that it was OK to remove here to make unfinished code more likely to compile while developing. I'm not wedded to removing it though.